data/json/npcs/TALK_NC_FARMER.json

```diff
@@ -0,0 +1,90 @@
+      { "text": "I don't have time for this.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_NC_FARMER_INTRO",
+    "dynamic_line": "Hey, you're alive! Me too. Been a long while since I've had a conversation. Uh, with someone else. Um.",
+    "responses": [
+      { "text": "It's not safe here anymore, you need to move on.", "topic": "TALK_NC_FARMER_LEAVE" },
+      { "text": "How did you make it?", "topic": "TALK_NC_FARMER_STORY1" },
+      { "text": "What did you do before the Cataclysm?", "topic": "TALK_NC_FARMER_EXPERTISE" },
+      { "text": "Have anything to trade?", "topic": "TALK_NONE", "effect": "start_trade" },
+      { "text": "I'm sorry, I gotta go.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_NC_FARMER_LEAVE",
+    "dynamic_line": "I heard it's bad in the cities, back when the radio was still working. Some of those things came through here. I want to stay out in the country if I can stay safe.",
+    "responses": [
+      { "text": "What did you do before the Cataclysm?", "topic": "TALK_NC_FARMER_EXPERTISE" },
+      { "text": "Wanna get outta here?", "topic": "TALK_SUGGEST_FOLLOW" },
+      { "text": "Let's trade items.", "topic": "TALK_NONE", "effect": "start_trade" },
+      { "text": "I'm sorry, I gotta go.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_NC_FARMER_STORY1",
+    "dynamic_line": "One of the nice things about the country life is the troubles of the world take a little longer to reach you, and some don't get you at all. We didn't see any dead the first day, but we heard all kinds of crazy things on the news.",
+    "responses": [
+      { "text": "What happened next?", "topic": "TALK_NC_FARMER_STORY2" },
+      { "text": "What did you do before the Cataclysm?", "topic": "TALK_NC_FARMER_EXPERTISE" },
+      { "text": "Let's trade items.", "topic": "TALK_NONE", "effect": "start_trade" },
+      { "text": "I'm sorry, I gotta go.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_NC_FARMER_STORY2",
+    "dynamic_line": "Come a little closer, I don't want to make too much noise. You smell much nicer than what I'm used to. Someone in trouble banged on the door that night, she said she was desperate, she had a baby, her home had burned down and her drugs were gone and there were monsters in the streets. We figured she was on drugs, but we couldn't turn them away so we gave them food and let them stay in the barn.",
+    "responses": [
+      { "text": "That's decent, was she all right?", "topic": "TALK_NC_FARMER_STORY3" },
+      { "text": "What did you do before the Cataclysm?", "topic": "TALK_NC_FARMER_EXPERTISE" },
+      { "text": "Let's trade items.", "topic": "TALK_NONE", "effect": "start_trade" },
+      { "text": "I'm sorry, I gotta go.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_NC_SURVIVOR_CHEF_STORY3",
```

Comment:
wrong ID? "TALK_NC_SURVIVOR_CHEF_STORY3"  I don't see it in any of the topics.

Reply:
Yep, thanks!